test/factories.rb

```diff
@@ -1,5 +1,10 @@
 # This will guess the User class
-FactoryGirl.define do
+FactoryGirl.define do  factory :user do
+    username "MyString"
+password_digest "MyString"
+role 1
+  end
```

Comment:
`end` at 6, 2 is not aligned with `factory :user do` at 2, 23 or `FactoryGirl.define do  factory :user do` at 2, 0.